ouroboros-network/protocol-tests/Test/ChainProducerState.hs

```diff
@@ -151,56 +150,56 @@ prop_switchFork (ChainProducerStateForkTest cps f) =
   in
       invChainProducerState cps'
       && all
-        (uncurry readerInv)
-        (zip (readerStates cps) (readerStates cps'))
+        (uncurry followerInv)
+        (zip (followerStates cps) (followerStates cps'))
   where
-    readerInv :: HasHeader block
-              => ReaderState block -> ReaderState block -> Bool
-    readerInv r r'
+    followerInv :: HasHeader block
+                => FollowerState block -> FollowerState block -> Bool
+    followerInv r r'
```

Comment:
Actually, this command

```
git grep -n -w -e r -- $(git diff 7160cfa8e HEAD --name-status | grep -E -e ^M | awk '{print $2}' ) | grep -i -e follow
```

finds the lines that have both the word `r` and the word "follow". That finds a handful of pretty clear cases. (There are some false positives in types too, but I think the `r` type variable in those hits is for "reference", not for "reader".)

Reply:
And this command similar finds occurrences of `rid` abbreviating "reader id".

```
git grep -n -w -e rid -- $(git diff 7160cfa8e HEAD --name-status | grep -E -e ^M | awk '{print $2}' )
```

Reply:
👍